spring-boot-project/spring-boot-dependencies/build.gradle

```diff
@@ -1107,6 +1107,24 @@ bom {
 			]
 		}
 	}
+	library("Pulsar", "3.0.0") {
+		group("org.apache.pulsar") {
+			modules = [
+					"pulsar-client",
+					"pulsar-client-admin",
+					"pulsar-client-all"
```

Comment:
I think this list of modules is incomplete. It should include every module that's released as a single "package".